controls/radchartview/features/hovermode.md

```diff
@@ -0,0 +1,86 @@
+---
+title: HoverMode
+page_title: HoverMode
+description: HoverMode
+slug: radchartview-features-hovermode
+tags: radchartview,hover, mode
+published: True
+position: 9
+---
+
+# HoverMode
+
+The different charts expose a property called **HoverMode** which controls weather the rest of the series will fade out when a certain series is hovered. In the case of the RadPieChart, all of the slices fade out except the hovered one. It is useful in situations with many overlapping series and has two possible values: **FadeOtherSeries** and **None**. The default value is **None**. This article will demonstrate this property with a RadCartesianChart.
```

Comment:
- "The different charts __from the RadChartView suite__ expose...."

- weather != whether

- you can move the information for the pie chart in a different paragraph at the end of the article. It could be a note.